spec/system/language_landing_page_spec.rb

```diff
@@ -0,0 +1,41 @@
+require 'rails_helper'
+
+describe 'Language Landing Page' do
+  before do
+    create(:article)
+    create(:article, :spanish)
+    create(:article, :german)
+    create(:article, :danish)
+    create(:article, :finnish)
+    create(:article, :french)
+    create(:article, :greek)
+    create(:article, :hebrew)
+    create(:article, :italian)
+    create(:article, :swedish)
+    create(:article, :turkish)
+    create(:article, :portuguese)
+    create(:article, :portuguese_br)
+  end
+
+  it 'has a link for every language' do
```

Comment:
This happens to be true at the moment. 

But it's possible to add a locale in the codebase but not have an article for it yet. That new locale should not show up on the /languages page with a zero count.

Does this test account for that?

Reply:
yes, but implicitly

in the test setup I create an article for every local and then assert `link_count == Article.count`

if you added a new locale, but without an article, the test would still pass.

I will add an explicit test for it

Reply:
@veganstraightedge adding that spec case paid off immediately :sweat_smile: 

https://github.com/crimethinc/website/pull/1425/files#diff-ce982c1bceed3c053a6aad28ecae4746R13